rm-datasource/src/main/java/com/alibaba/fescar/rm/datasource/undo/mysql/MySQLUndoDeleteExecutor.java

```diff
@@ -42,6 +43,7 @@ public MySQLUndoDeleteExecutor(SQLUndoLog sqlUndoLog) {
 
     @Override
     protected String buildUndoSQL() {
+        MySQLKeywordChecker mySQLKeywordChecker=new MySQLKeywordChecker();
```

Comment:
why not KeywordChecker type？

Reply:
Thanks for your carefully review,i will improve this by using a Factoryt ype.